tests/pages/services/ServiceActions-cy.js

```diff
@@ -220,10 +217,8 @@ describe("Service Actions", function() {
         });
         cy
           .get(".confirm-modal .button-collection .button-danger")
-          .as("dangerButton")
-          .click();
```

Comment:
Why are we dropping this test? With current changes test description `reenables button after faulty request` doesn't reflect on what it actually does.